Sources/StreamChat/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -143,6 +143,103 @@ class ChannelController_Tests: StressTestCase {
         XCTAssertEqual(Set(controller.messages.map(\.id)), Set(payload.messages.map(\.id)))
     }
 
+    // MARK: - Channel config feature tests
+    
+//    func test_markRead_error_if_disabled() throws {
+//        let payload = dummyPayload(with: channelId, channelConfig: ChannelConfig(readEventsEnabled: false))
+//        try client.databaseContainer.writeSynchronously { session in
+//            try session.saveChannel(payload: payload)
+//        }
+//    }
+    
+    func test_features_nil_channel() {
+        XCTAssertFalse(controller.areReactionsEnabled)
+        XCTAssertFalse(controller.areRepliesEnabled)
+        XCTAssertFalse(controller.areUploadsEnabled)
+        XCTAssertFalse(controller.areTypingEventsEnabled)
+        XCTAssertFalse(controller.areReadEventsEnabled)
+    }
+
+    func test_feature_read_events_enabled() throws {
```

Comment:
nit: We try to follow this naming structure (at least for the new tests):
![](https://qualitycoding.org/wp-content/uploads/2020/04/TestNaming-big.png)